packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -231,6 +232,17 @@ class CreateCommand extends CreateBase {
     }
 
     final String dartSdk = globals.cache.dartSdkBuild;
+    final bool includeIos = featureFlags.isIOSEnabled && platforms.contains('ios');
+    String developmentTeam;
```

Comment:
How about init it with '', so later, we only need to check `if (iosDevelopmentTeam.isNotEmpty)`, it would also fit better with NNBD, and make the migration easier. 

And nits: maybe iosDevelopmentTeam would be a better name as it was only used for iOS.

Reply:
I'd prefer if we left it as uninitialized, so we can differentiate between explicitly set as an empty string and never set.

Reply:
Is there a specific meaning of development team explicitly set as empty string? @jmagman Might be able to answer that. I was looking at the line below `'iosDevelopmentTeam': iosDevelopmentTeam ?? '',` thinking that they should be the same.